packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -215,7 +234,8 @@ export const Stepper = React.forwardRef<HTMLDivElement, StepperProps>(
       return () => window.removeEventListener('resize', handleResize);
     }, [windowWidth]);
 
-    // Step states
+    const stepLabels = React.Children.toArray(children);
```

Comment:
These aren't the labels, this is an array of the Stepper children

Reply:
This could be updated to something like below to avoid checking `React.Children.toArray(children)` twice
```
    const stepperChildren = React.Children.toArray(children);
    const numberOfSteps = stepperChildren.length;
```